time.c

```diff
@@ -3487,18 +3487,23 @@ time_s_mktime(int argc, VALUE *argv, VALUE klass)
  *  call-seq:
  *    to_i -> integer
  *
- *  Returns the number of seconds since the Epoch
- *  for the time represented in +self+:
+ *  Returns the value of +self+ as integer
+ *  {Epoch seconds}[rdoc-ref:Time@Epoch+Seconds];
+ *  subseconds are omitted (not rounded):
  *
- *    Time.utc(1970, 1, 1).to_i # => 0
- *    t = Time.now.to_i         # => 1595263289
+ *    Time.utc(1970, 1, 1, 0, 0, 0).to_i         # => 0
+ *    Time.utc(1970, 1, 1, 0, 0, 0, 999999).to_i # => 0
+ *    Time.utc(1950, 1, 1, 0, 0, 0).to_i         # => -631152000
+ *    Time.utc(1990, 1, 1, 0, 0, 0).to_i         # => 631152000
  *
- *  Subseconds are omitted:
+ *  Time#tv_sec is an alias for Time#to_i.
  *
- *    t = Time.now # => 2022-07-12 09:13:48.5075976 -0500
- *    t.to_i       # => 1657635228
+ *  Related:
```

Comment:
I would remove `Time.at`.  Since `to_f` and `to_r` are instance methods and not class methods, you should use `#to_f` and `#to_r` instead of `Time.to_f` and `Time.to_r`.  I don't think there is significant value in describing the difference between `to_f` and `to_r` here.  I would use: `Related: #to_f, #to_r`.  Similar change for `to_f` and `to_r` method documentation.

Reply:
Fixed.